stable/redis-ha/values.yaml

```diff
@@ -138,3 +138,7 @@ persistentVolume:
   annotations: {}
 init:
   resources: {}
+
+## Use hostPath for data (requires persistentVolume.enabled to be false)
```

Comment:
Could you add a matching line in the README.md `values` table as well?

Reply:
Done.